ansible/files/prometheus/prometheus.yml

```diff
@@ -21,6 +21,58 @@ scrape_configs:
         - 'tradedefecit.conference.fosdem.net:9100'
         - 'prometheus.conference.fosdem.net:9100'
 
+  - job_name: 'video_node'
```

Comment:
Just call these 'node', put them in with the others.

Are you missing a . before your v?